docs/docsite/rst/porting_guides/porting_guide_2.6.rst

```diff
@@ -22,7 +22,8 @@ No notable changes.
 Deprecated
 ==========
 
-No notable changes.
+* In the `nxos_igmp_interface` module, `oif_prefix` and `oif_source` properties are deprecated. Use
```

Comment:
We can make `nxos_igmp_interface` a link, see  porting_guide_2.5.rst for an example.